test-cases/performance/perf-regression-latency-lwt-int.yaml

```diff
@@ -0,0 +1,29 @@
+test_duration: 300
+prepare_write_cmd: [
+  "cassandra-stress user profile=/tmp/cs_lwt_perf_int.yaml 'ops(update=1)' no-warmup cl=QUORUM n=2000000 -port jmx=6868 -mode cql3 native -rate threads=16"
+]
+
+stress_cmd_lwt_ine: "cassandra-stress user profile=/tmp/cs_lwt_perf_int.yaml 'ops(insert-if-not-exists=1)' no-warmup cl=QUORUM duration=10m -port jmx=6868 -mode cql3 native -rate threads=16"
+stress_cmd_lwt_uc: "cassandra-stress user profile=/tmp/cs_lwt_perf_int.yaml 'ops(update-if-cond=1)' no-warmup cl=QUORUM duration=10m -port jmx=6868 -mode cql3 native -rate threads=16"
+stress_cmd_lwt_ue: "cassandra-stress user profile=/tmp/cs_lwt_perf_int.yaml 'ops(update-if-exists=1)' no-warmup cl=QUORUM duration=10m -port jmx=6868 -mode cql3 native -rate threads=16"
+stress_cmd_lwt_de: "cassandra-stress user profile=/tmp/cs_lwt_perf_int.yaml 'ops(delete-if-exists=1)' no-warmup cl=QUORUM duration=10m -port jmx=6868 -mode cql3 native -rate threads=16"
+
```

Comment:
What is the load on Scylla in terms of CPU when running these workloads with rate threads = 16?
When testing latency we test when the server isn't loaded to 100% and we usually want to control the throughput.
consider using "throttle" (see other latency tests)

Reply:
Thanks, that will be done shortly.